### ISSUE TITLE
Does XLA handle FusedBatchNormV3 correctly in GPUs

### ISSUE THREAD

**Issue by whatdhack** [tensorflow/tensorflow#52668]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below):: 2.6
- Python version: 3.8
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 11.4
- GPU model and memory:

**Describe the current behavior**
It appears that when tf_xla_auto_jit=2 is set and mixed precision (fp16) is used,  networks run  significantly faster (3.5x) over non-XLA .  The advantage disappears when batchnorm is disabled ( about 1.3 x) 

**Describe the expected behavior**
Should not show such marked differences  with and without batchnorm.

**[Contributing](https://www.tensorflow.org/community/contribute)**

- Do you want to contribute a PR? (yes/no):
- Briefly describe your candidate solution(if contributing):

**Standalone code to reproduce the issue**
Just use any code  with tf.keras.layers.BatchNormalization  

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.
